cEP-0019.md

```diff
@@ -0,0 +1,208 @@
+emojis. Those emojis are called meta-review, or
+[reactions](https://developer.github.com/v4/enum/reactioncontent/). THUMBS_UP
+and THUMBS_DOWN, together with other reactions, are used by people especially
+the author of pull request to publicly acknowledge or reject review comments.
+This provides feedback to reviewers and all other members.
+
+However, those reactions spread over all PRs and are not collected and
+analyzed. By tracking reactions, a nice feedback loop would be created
+for both senior and junior reviewers.
+
+Introduction
+------------
+
+The meta-review project is about building a meta-review system,
+similar to a meta-moderation system, that can track emoji responses to
+review comments. These responses are to be collected, processed and displayed
+on [gh-board](https://github.com/coala/gh-board), which is a nice serverless
+kanban board.
+
+The scope of this project would exclude non-public meta-reviews. That is,
+there would not be any anonymous meta-reviews as that introduces too many
+additional complexities. The public only version would be used for
+a long time before adding support for non-public meta-reviews.
+
+Objectives
+----------
+
+One goal of this project is to encourage people to do meta-reviews,
+systematically.
+
+Also, to encourage people to do more meta-reviews, statistics of meta-reviews
+are to be collected and analysed. People who do meta-reviews will get scores
+according to some metrics. A ranking list is to be displayed on gh-board.
+
+To conclude, this project builds a meta-review system on gh-board to display
+statistics (score ranking, meta-reviews in need), and encourage people to
+do meta-review.
+
+Proposed Approach
+-----------
+
+### Collection of Meta-reviews
+
+Meta-review info, namely reactions, can be fetched via GitHub API V4 GraphQL.
+A Sample query is as follows:
+
+```GraphQL
+query {
+  repository(owner:"coala", name:"coala-bears") {
+    pullRequest(number:2372) {
+      author {
+        login
+        url
+      }
+      comments(first:10) {
+        nodes {
+          author {
+            login
+            url
+          }
+          reactions(first:10) {
+            totalCount
+            nodes {
+              content
+              user {
+                login
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The collection of meta-reviews includes two parts:
+
+##### Pre-fetch meta-reviews
+
+Currently, `fetch-issues.js` pre-fetches issues and pull requests to speed up
+gh-board performance and save users' API hits. Meta-reviews are also to be
+fetched during this procedure.
+
+##### Fetch meta-reviews
+
+When a new issue/pull request is detected, gh-board will fetch that issue and
+loads it asynchronously. Meta-reviews also need to be fetched during this
+procedure.
+
+### Analysis of Meta-reviews
+
+This paragraph explains how meta-review information can be utilized to
```

Comment:
> ... paragraph ...

That doesnt make sense here, as this paragraph has only one sentence and it doesnt explain anything.

Reply:
Should be `section` instead of `paragraph`. 😅